src/move.c

```diff
@@ -173,6 +173,7 @@ char MoveClient(ClientNode *np, int startx, int starty)
          /* Determine if we are at a border for desktop switching. */
          sp = GetCurrentScreen(np->x + np->width / 2, np->y + np->height / 2);
          atLeft = atTop = atRight = atBottom = 0;
+         MaxFlags flags = MAX_NONE;
```

Comment:
It's probably unnecessary these days, but JWM mostly has variables declared at the beginning of blocks rather than where they are needed.  Being able to mix declarations and code is a C99 thing.